Tests/CommandsTests/BuildToolTests.swift

```diff
@@ -150,7 +150,7 @@ final class BuildToolTests: CommandsTestCase {
             do {
                 let result = try build(["--product", "exec1"], packagePath: fullPath)
                 XCTAssertMatch(result.binContents, ["exec1"])
-                XCTAssertNoMatch(result.binContents, ["exec2.build"])
+                XCTAssertMatch(result.binContents, ["exec2.build"])
```

Comment:
these test changes make me worried? seems like we are also changing the logic, not just the use of a file?

Reply:
The logic is the same - the differences here are because `exec2.build` is a directory, and that directory will now exist because we wrote out the file list.